webservices/rest.py

```diff
@@ -43,6 +43,23 @@ def sqla_conn_string():
 api = restful.Api(app)
 db.init_app(app)
 
+# api.data.gov
+trusted_proxies = ('192.168.114.100',)
+PRODUCTION = os.getenv('PRODUCTION', False)
+
+
+@app.before_request
+def limit_remote_addr():
+    if PRODUCTION != False:
```

Comment:
Unfortunately this will still fail in case somebody sets PRODUCTION=False, because False will come in from `os.getenv()` as the string 'False', and 'False' evaluates to True in a boolean context.

    >>> bool('False')
    True

I'd recommend doing something like this:

http://stackoverflow.com/a/715468